### PR TITLE
Added GitHub pages content

### PR DESCRIPTION
**Pull request by fhubi**
Added the following mdx files:
- alternatives.mdx
- background.mdx
- getting-started.mdx
- hero.mdx
- use-cases.mdx
- documentation-hero.mdx
- examples.mdx

### CODE REVIEW

File: site/content/background.mdx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+description: Background and origin story of the project
+title: Background
+---
+
+ComposeUI is a dotnet core based, WebView2 using UI Container for hybrid web-desktop applications.
```
Comment (ztanczos) [inline site/content/background.mdx:6]: I'd leave out 'core', just say .NET based

File: site/content/documentation/examples.mdx [hunk 2]
```diff
@@ -0,0 +1,19 @@
+---
+order: 1
+title: Examples
+---
+
+### FCD3 App Directory example
+
+This example contains a JSON file that can be used as the data source for ComposeUI's FDC3 App Directory implementation. The App Directory contains the [js-chart](https://github.com/fhubi/ComposeUI/tree/main/examples/js-chart) and [js-grid](https://github.com/fhubi/ComposeUI/tree/main/examples/js-datagrid) examples.
```
Comment (ztanczos) [inline site/content/documentation/examples.mdx:8]: This is your personal fork, not the main repo ;)

File: site/content/background.mdx (hunk 1 above)
Comment (psmulovics) [reply site/content/background.mdx:6]: Should we say .NET instead of dotnet?

Comment (fhubi) [reply site/content/background.mdx]: Changed dotnet core to .NET

File: site/content/documentation/examples.mdx (hunk 2 above)
Comment (fhubi) [reply site/content/documentation/examples.mdx]: Fixed.

**Review by psmulovics — DISMISSED** [review]

File: site/content/hero.mdx [hunk 3]
```diff
@@ -0,0 +1,17 @@
+---
+description: Featured content hero
+title: Hero
+---
+
+<Hero title="Crossroads">
```
Comment (kruplm) [inline site/content/hero.mdx:6]: Hero title should be "ComposeUI" instead.

**Review by kruplm — CHANGES_REQUESTED** [review]

File: site/content/hero.mdx (hunk 3 above)
Comment (fhubi) [reply site/content/hero.mdx]: Fixed.

**Review by kruplm — APPROVED** [review]

**Review by ztanczos — APPROVED** [review]

**Pull request merged**